examples/chef/linux/main.cpp

```diff
@@ -29,6 +29,14 @@ using namespace chip::Shell;
 using namespace chip::app;
 using namespace chip::app::Clusters;
 
+#if CHIP_CONFIG_ENABLE_ICD_CIP
+void notifyIcdActive(System::Layer * layer, void *)
+{
+    ICDNotifier::GetInstance().NotifyNetworkActivityNotification();
+    DeviceLayer::SystemLayer().StartTimer(chip::System::Clock::Milliseconds32(10000), notifyIcdActive, nullptr);
+}
```

Comment:
10s is not representative of real ICD devices.

Reply:
Yes, we need clear comments here, and make it runtime configurable. thanks

Reply:
It is set as configurable with default 60 seconds now. PTAL